helpers/lib/fonts.js

```diff
@@ -87,8 +87,16 @@ module.exports = function(format, themeSettings, handlebars, options) {
     
     const collectedFonts = {};
     _.each(themeSettings, function(value, key) {
-        if (!fontKeyFormat.test(key)) {
+        //check that -font is on end of string but not start of string
```

Comment:
I think you are probably safe to use `key.endsWith('-font')`

Reply:
fixed.